loadable_apps/loadable_sample/wifiapp/Makefile

```diff
@@ -61,6 +61,10 @@ SRCS += recovery.c
 endif
 ifeq ($(CONFIG_EXAMPLES_BINARY_UPDATE_TEST),y)
 SRCS += binary_update.c
+else
```

Comment:
Do you mean `BINARY_UPDATE_TEST` is y or `UPDATE_AGING_TEST` is y?
If then, you can use `filter` instead of current duplicated condition.
`ifeq ($UPDATE_AGING_TEST),$(filter $(UPDATE_AGING_TEST),y y)`